woocommerce/woocommerce
# Use separate Travis build jobs to run e2e tests and PHPCS

### Changes proposed in this Pull Request:

This PR creates two new Travis build jobs, one to run the e2e tests and another to run PHPCS. Doing this, instead of running those two checks in the same build job as the PHP 7.2 unit tests, should make it easier to see why a given build failed and it should make the total build time shorter as PHPCS, e2e tests and unit tests will run in parallel.

### How to test the changes in this Pull Request:

1. Make sure that all build jobs pass and that they execute what they are supposed to execute.
